khal/ui/__init__.py

```diff
@@ -1344,7 +1344,12 @@ def emit(self, record):
     palette = _add_calendar_colors(
         getattr(colors, pane._conf['view']['theme']), pane.collection)
     loop = urwid.MainLoop(
-        frame, palette, unhandled_input=frame.on_key_press, pop_ups=True)
+        widget=frame,
+        palette=palette,
```

Comment:
I took the liberty to use kwargs instead of args, to make the urwid API call clearer